lib/zipkin-tracer/config.rb

```diff
@@ -39,8 +39,6 @@ def initialize(app, config_hash)
       if @sampled_as_boolean
         @logger && @logger.warn("Using a boolean in the Sampled header is deprecated. Consider setting sampled_as_boolean to false")
       end
-      # Record the given tags on server receive, even if the zipkin headers were present in the incoming request?
-      @record_on_server_receive = parse_tags(config[:record_on_server_receive])
```

Comment:
you can remove [`:record_on_server_receive` on line 11](https://github.com/openzipkin/zipkin-ruby/pull/144/files#diff-783b62518b0223696f3cc8e6c120d4e4R11)